lib/log/main.go

```diff
@@ -54,9 +65,55 @@ func init() {
 		panic("Unable to create a logger")
 	}
 
-	Log = RawLog.Sugar()
+	Log = sentryInit(RawLog).Sugar()
+}
+
+func sentryInit(log *zap.Logger) *zap.Logger {
+	errConfig := config.ParseEnv()
```

Comment:
surely this is already called somewhere else in the code... is it idempotent? Should we be initing the config here and passing it through?

Reply:
It's not, since this is new code -- it's a private function inside the log module, so it won't be accessible elsewhere.  It's broken out a separate function to keep the sentry stuff separate and more readable (at least that was the idea)

Reply:
It is idempotent, but I did consider passing it through.  It's just that it would touch a lot more code to do so -- not sure the best solution here